inc/datahandlers/user.php

```diff
@@ -1536,17 +1536,24 @@ function delete_content($delete_uids=false)
 		$plugins->run_hooks('datahandler_user_delete_content', $this);
 
 		$db->delete_query('userfields', "ufid IN({$this->delete_uids})");
-		$db->delete_query('privatemessages', "uid IN({$this->delete_uids})");
+		$db->delete_query('privatemessages', "uid IN({$this->delete_uids}) OR fromid IN({$this->delete_uids})");
```

Comment:
@Destroy666x If you delete inactive users you may want to keep the PMs they sent when they were active. It doesn't delete posts and threads.